src/AddTodoForm.jsx

```diff
@@ -22,11 +23,7 @@ export const AddTodoForm = ({ onAddTodo }) => {
   return (
     <div className="text-bg-dark container-lg">
       <form className="m-5 pb-1" onSubmit={handleAddTodo}>
-        <label htmlFor={todoTitle} className="h5 m-1 mt-3">
-          Title
-        </label>
-        <input name="title" type="text" id={todoTitle} value={todoTitle} onChange={handleTitleChange} />
-        <input type="submit" className="btn btn-primary m-1 mb-2" value="Add" />
+        <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange} isFocused> title </InputWithLabel>
```

Comment:
On line 18, great destructuring! But it looks like those variables aren't actually being used in the code, so it's best practice to remove those for now.